src/sentry/monitors/utils.py

```diff
@@ -18,3 +21,51 @@ def signal_first_monitor_created(project: Project, user, from_upsert: bool):
         first_cron_monitor_created.send_robust(
             project=project, user=user, from_upsert=from_upsert, sender=Project
         )
+
+
+def get_alert_rule(project: Project, user: User, monitor: Monitor, alert_rule: dict):
```

Comment:
Could we add a doc comment here to describe what the expected input of `alert_rule` is?